libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -287,44 +306,59 @@ void AP_MotorsHeli_RSC::output(RotorControlState state)
             _governor_fault = false;
             if (_in_autorotation) {
                 // if in autorotation and using an external governor, set the output to tell the governor to use bailout ramp
-                _control_output = constrain_float( _rsc_arot_bailout_pct/100.0f , 0.0f, 0.4f);
+                _idle_throttle = constrain_float( _ext_gov_arot_pct/100.0f , 0.0f, 0.4f);
+                if(!_autorotating){
```

Comment:
nitpick: should have space between if and "("

Reply:
fixed.  Thanks!